src/main/scala/zio/prelude/Bicovariant.scala

```diff
@@ -0,0 +1,56 @@
+package zio.prelude
+
+trait RightMap[:=>[_,+_]] {
```

Comment:
Even just the name suggests that there should be a complementary `LeftContravariant`. WDYT?

Reply:
Indeed it is possible to have LeftContravariant and LeftCovariant but there are not so many instances for those and only RightCovariant is reusable. OTOH using those 3 abstractions gives a nice split of laws.

I explored this idea and pushed it a lot further in my library, if you want to take a 👀  : https://github.com/lemastero/Triglav/blob/master/src/main/scala/Triglav/face2/Bifunctor.scala.

But lets start with checking if Bifunctor is of interest to zio-prelude :)

Reply:
🃏 :clown_face:
:smile:

Reply:
LeftContravariant will be added in later PR as:
`trait TriLeftCovariant[Z[_, +_, _]]`
`trait TriLeftMostCovariant[Z[+_, _, _]]`
ATM there is no reason to add it when we have 2 type params.

BTW RightCovariant will be replaced by [_,_,+_] version.